components/drawer/demo/loading.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import type { DrawerProps } from 'antd';
+import { Button, Drawer } from 'antd';
+
+const App: React.FC = () => {
+  const [open, setOpen] = useState(false);
+  const [loading] = useState<DrawerProps['loading']>(true);
+
+  const showDrawer = () => {
+    setOpen(true);
+  };
+
+  const onClose = () => {
+    setOpen(false);
+  };
+
+  return (
+    <>
+      <Button type="primary" onClick={showDrawer}>
+        Open
+      </Button>
+      <Drawer
+        title="Basic Drawer"
+        placement="right"
+        closable={false}
+        onClose={onClose}
+        open={open}
+        loading={loading}
```

Comment:
How about adding a nprogress loading UI for Drawer?

https://ricostacruz.com/nprogress/

```jsx
loading: boolean | { progress?: number };
```

Reply:
@yoyo837 in my experience spinner look more appropriate.

Reply:
I could play with, one question, does this library [nprogress](https://ricostacruz.com/nprogress/) already installed?